.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,73 @@
+name: API Performance Benchmarks
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: "0 12 * * *"
+
+jobs:
+
+  macos:
+    name: macOS ${{ matrix.python }} + ${{ matrix.version }}
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ['3.8']
+        version: ['tensorflow==2.4.0:tensorflow-io-nightly', 'tensorflow==2.4.0:tensorflow-io']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Setup macOS
+        run: |
+          set -x -e
+          python -m pip install -U wheel setuptools
+          python --version
+      - name: Test macOS
+        run: |
+          set -x -e
+          python --version
+          df -h
+          rm -rf tensorflow_io
+          echo ${{ matrix.version }} | awk -F: '{print $1}' | xargs python -m pip install -U
+          echo ${{ matrix.version }} | awk -F: '{print $2}' | xargs python -m pip install --no-deps -U
+          python -m pip install pytest-benchmark scikit-image
+          python -m pip freeze
+          python -c 'import tensorflow as tf; print(tf.version.VERSION)'
+          python -c 'import tensorflow_io as tfio; print(tfio.version.VERSION)'
+          python -m pytest --benchmark-only -v --import-mode=append $(find . -type f \( -iname "test_*_eager.py" ! \( -iname "test_bigquery_eager.py" \) \))
```

Comment:
why is `test_bigquery_eager.py` handled differently?

Reply:
@burgerkingeater I remember that was due to https://github.com/grpc/grpc/issues/20034